src/common/Constants.js

```diff
@@ -86,6 +85,26 @@ define(['common/core/constants', 'common/storage/constants'], function (CORE, ST
             RENAME_META_POINTER_TARGET: 'renameMetaPointerTarget',
             CHANGE_ASPECT_META: 'changeAspectMeta',
             REMOVE_META_RULE: 'removeMetaRule'
+        },
+
+        MULTILINE_MODE: {
```

Comment:
`ATTRIBUTE_MULTILINE_TYPES`
Also extend the core-docs with examples of the new "properties" `getAttributeMeta`